src/components/Title.jsx

```diff
@@ -1,7 +1,21 @@
 import React from "react";
 
+// const Title = ({ title }) => {
+//   return <div>{title}</div>;
+// };
+
+// export default Title;
 const Title = ({ title }) => {
-  return <div>Title</div>;
+  return (
+    <div className="flex-col inline-flex gap-3 items-center justify-center">
```

Comment:
you only need 1 div  you can remove the rest as well as make sure you use ais-blue-100 for the text color.

Reply:
Which div do you want me to keep? Every time I remove a div, the code stops working.

Reply:
all the them 
you just to have you one div so like 
<div (styling)> {title} <div>